admin.py

```diff
@@ -22,6 +22,7 @@
 admin.add_view(MerchantTxRestrictedModelView(MerchantTx, db.session, category='Reports'))
 admin.add_view(SettlementAdminModelView(Settlement, db.session, category='Reports'))
 admin.add_view(SettlementRestrictedModelView(Settlement, db.session, endpoint='SettlementUser', category='Reports'))
+admin.add_view(BankRestrictedModelView(Bank, db.session))
```

Comment:
put this in the 'Users' category, group it with the others in that category

Reply:
Added to users categorry and rearranged admin.py